engine/src/test/java/org/camunda/bpm/engine/test/api/multitenancy/query/history/MultiTenancyHistoricExternalTaskLogTest.java

```diff
@@ -271,14 +288,14 @@ public void testGetErrorDetailsNoAuthenticatedTenants() {
     } catch (ProcessEngineException e) {
       // then
       String errorMessage = e.getMessage();
-      assertThat(errorMessage.contains("Cannot get the historic external task log "), is(true));
-      assertThat(errorMessage.contains(failedHistoricExternalTaskLogId), is(true));
-      assertThat(errorMessage.contains("because it belongs to no authenticated tenant."), is(true));
+      assertThat(errorMessage.contains("Cannot get the historic external task log ")).isEqualTo(true);
```

Comment:
No need to change it, but fyi there are .isTrue() and .isFalse() methods that are a bit shorter and do the same thing.